courses/migrations/0033_populate_letter_grades.py

```diff
@@ -4,16 +4,22 @@
 import pytz
 from django.db import migrations
 
+from courses.models import Program
 from courses.utils import convert_to_letter
 
 
 def populate_letter_grade(apps, schema_editor):
     """Populate the certificate_available_date from course run's end_date"""
```

Comment:
I know it wasn't part of this PR, but could you update this docstring to match the function?